library/core/src/iter/traits/iterator.rs

```diff
@@ -3057,6 +3058,48 @@ pub trait Iterator {
         Cycle::new(self)
     }
 
+    /// Returns an iterator over all contiguous windows of length `N`. The
+    /// windows overlap. If the iterator is shorter than `N`, the iterator
+    /// returns no values.
+    ///
+    /// `array_windows` clones the iterator elements so that they can be part of
+    /// successive windows, this makes this it most suited for iterators of
+    /// references and other values that are cheap to clone.
+    ///
+    /// # Panics
+    ///
+    /// If called with `N = 0`.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// #![feature(iter_array_windows)]
+    ///
+    /// let mut iter = "rust".chars().array_windows();
```

Comment:
```suggestion
    /// let mut iter = "rust".chars().array_windows::<2>();
```
I think you missed it?

Reply:
It's not required because of type inference